### PR TITLE
Optimizing docker build

### PR DESCRIPTION
**Pull request by yann300**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,22 @@ npm run setupremix  # only if you plan to link remix and remix-ide repositories
...
+Prerequisites: 
+* Docker (https://docs.docker.com/desktop/)
+* Docker-compose (https://docs.docker.com/compose/install/)
+
+### Run with docker:
+
+To run locally without building you only need docker-compose.yaml file and you can run:
+
+```
+docker-compose up -d
```
Comment (yann300) [inline README.md:59]: maybe worth saying that this command will run the lastest code from master.
Also that is it possible to pull the docker without cloning the repo:
for running master branch
`
docker pull remixproject/remix-ide:master
docker run -p 8080:80 remixproject/remix-ide:master
` 

for running the release branch:
`
docker pull remixproject/remix-ide:remix_live
docker run -p 8080:80 remixproject/remix-ide:remix_live
`

Comment (edisinovcic) [reply README.md:59]: Just running

```
docker pull remixproject/remix-ide
```
or
```
docker pull remixproject/remix-ide:latest
```

will fetch from master.

Comment (yann300) [reply README.md:59]: yeah note that `latest` here is not the latest release but the code in master. The latest release is at `remix_live`

Comment (edisinovcic) [reply README.md:59]: Yes. Will update readme with that.

Comment (edisinovcic) [reply README.md:75]: Updated.

**Pull request merged**